src/popups/info.css

```diff
@@ -115,3 +115,9 @@ input {
 	cursor: default !important;
 	text-decoration: none !important;
 }
+
+.artist, .album, .track {
+	overflow: hidden;
```

Comment:
The similar block is already defined on line 73:
```css
.artist, .album {
	overflow: hidden;
	text-overflow: ellipsis;
	white-space: nowrap;
}
```

I suggested to update that one to add `.track` class.